package.json

```diff
@@ -29,17 +29,17 @@
   },
   "homepage": "https://github.com/braintree/asset-loader#readme",
   "devDependencies": {
-    "@types/jest": "^29.4.0",
+    "@types/jest": "^29.5.3",
     "@types/promise-polyfill": "^6.0.4",
-    "@typescript-eslint/eslint-plugin": "^5.54.1",
-    "eslint": "^8.35.0",
+    "@typescript-eslint/eslint-plugin": "^5.62.0",
+    "eslint": "^8.47.0",
     "eslint-config-braintree": "^6.0.0-typescript-prep-rc.2",
-    "eslint-plugin-prettier": "^4.2.1",
-    "jest": "^29.5.0",
-    "jest-environment-jsdom": "^29.5.0",
-    "prettier": "^2.8.4",
-    "ts-jest": "^29.0.5",
-    "typescript": "^4.9.5"
+    "eslint-plugin-prettier": "^5.0.0",
+    "jest": "^29.6.3",
+    "jest-environment-jsdom": "^29.6.3",
+    "prettier": "^3.0.2",
+    "ts-jest": "^29.1.1",
+    "typescript": "^5.1.6"
```

Comment:
just to be on the safe side, can you confirm there are no breaking changes from typescript v4 to v5 that would kill support for older integrations? I just want to be sure since this is used by braintree-web

Reply:
https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#breaking-changes-and-deprecations

2 main points:

- It now requires a newer version of node, which we were already in compliance with.
- TS v5 now targets ES2018, but we can still choose to compile to ES5.

_If_ there are any breaking changes that we have to introduce during the modernisation of these libraries, we'll be sure to make the necessary changes in braintree-web (like changing the imports/exports, etc.)

Reply:
To add on to Oscar's comment, older versions of the braintree-web SDK would be using older versions of the asset-loader. So we shouldn't run into any issues with these changes breaking older integrations.

Reply:
ok great, thanks for confirming!